### PR TITLE
jewel: rbd: partition func should be enabled When load nbd.ko for rbd-nbd

### PR DESCRIPTION
**Pull request by Abhishekvrshny**
http://tracker.ceph.com/issues/18285

### CODE REVIEW

**Comment by smithfarm** [comment]
@trociny This PR passed an RBD suite at http://tracker.ceph.com/issues/17851#note-41 
with a single failure [1] that was not reproducible in a subsequent run.

[1] The failure was a reoccurrence of http://tracker.ceph.com/issues/17695
which was supposed to have been fixed in jewel by https://github.com/ceph/ceph/pull/11644

File: src/tools/rbd_nbd/rbd-nbd.cc [hunk 1]
```diff
@@ -62,14 +62,18 @@ static void usage()
             << "Options:\n"
             << "  --device <device path>                    Specify nbd device path\n"
             << "  --read-only                               Map readonly\n"
-            << "  --nbds_max <limit>                        Override for module param\n"
+            << "  --nbds_max <limit>                        Override for module param nbds_max\n"
+            << "  --max_part <limit>                        Override for module param max_part\n"
+            << "  --exclusive                               Forbid other clients write\n"
```
Comment (trociny) [inline src/tools/rbd_nbd/rbd-nbd.cc]: @Abhishekvrshny `--exclusive` option wrongly leaked to this commit

File: src/tools/rbd_nbd/rbd-nbd.cc [hunk 2]
```diff
@@ -62,14 +62,18 @@ static void usage()
...
             << std::endl;
   generic_server_usage();
 }
 
 static std::string devpath, poolname("rbd"), imgname, snapname;
 static bool readonly = false;
 static int nbds_max = 0;
+static int max_part = 255;
+static bool exclusive = false;
```
Comment (trociny) [inline src/tools/rbd_nbd/rbd-nbd.cc]: And here too.

**Comment by Abhishekvrshny** [comment]
@trociny : Made the required changes. Please review

**Comment by dachary** [comment]
passed rbd suite at http://tracker.ceph.com/issues/17851#note-76

**Pull request merged**